### PR TITLE
added some links and fixed the Carousel Comments Slide for mobile

### PR DESCRIPTION
**Pull request by benjithorpe**
Annotation 2020-08-13 035313

### CODE REVIEW

File: Useful-links/Useful-links.html [hunk 1]
```diff
@@ -24,126 +22,131 @@ <h3>Useful Links To Share</h3>
...
-      <li>.</li> <!-- Share your information here -->
-      <li>.</li> <!-- Share your information here -->
-    </ul>
-  </div>
+    <div class="list">
+      <h3>Websites to learn for free</h3>
+      <ul>
+        <li><a href="https://www.freecodecamp.org/" target="_blank">Freecodecamp</a> Learn for free</li>
+        <li><a href="https://www.khanacademy.org" target="_blank">Khan Academy</a> Learn for free</li>
+        <li><a href="https://learntocodewith.me/posts/code-for-free/" target="_blank">learntocodewithme</a> 81 websites to learn for free</li>
```
Comment (SyntappZ) [inline Useful-links/Useful-links.html:75]: this looks like a great resource for learning code

**Comment by SyntappZ** [comment]
@benjithorpe 

[81 websites to learn for free](https://learntocodewith.me/posts/code-for-free/ ) this looks like a great resource thanks for adding that, i have fixed the merge conflicts but it looks like you forgot to checkout a branch and pushed your master

File: index.html [hunk 2]
```diff
@@ -10,7 +10,16 @@
     href="https://use.fontawesome.com/releases/v5.4.1/css/all.css">
     <link rel='stylesheet' href='assets/css/style.css' type='text/css'>
     <!-- Bootstrap CDN-->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" 
+      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+      <style>
+        @media screen and (max-width: 550px) {
+          #randSec h2 {
```
Comment (DavidMatthewFraser) [inline index.html:18]: This does not target the text inside the slides.
You can accomplish this with:
```
@media screen and (max-width: 550px) {
          .rev_div > p {
            font-size: 22px;
          }
        }
```
which targets the children of all all divs of class `.rev_div` (which is where all the comments go).
However, after testing the above change, the comment slide looks much better!
I will add these changes to main.css

File: index.html [hunk 3]
```diff
@@ -10,7 +10,16 @@
     href="https://use.fontawesome.com/releases/v5.4.1/css/all.css">
     <link rel='stylesheet' href='assets/css/style.css' type='text/css'>
     <!-- Bootstrap CDN-->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" 
+      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+      <style>
```
Comment (DavidMatthewFraser) [inline index.html:16]: We use an external css file for the styling of this page for readability, I will move your changes there.

File: challenges/challenge1.html [hunk 4]
```diff
@@ -103,7 +103,7 @@ <h1>Benjamin Thorpe</h1>
               <p><q>Stay Hungry, Stay Foolish</q> - Steve Jobs</p>
             </div>
 
-            <!-- avatar for Benjamin tfrick47 -->
+            <!-- avatar for Terri Fricker -->
```
Comment (DavidMatthewFraser) [inline challenges/challenge1.html:106]: Good Catch!

File: challenges/challenge2.html [hunk 5]
```diff
@@ -37,13 +40,15 @@ <h1 id='title' class='text-center'>Open-Source Solar System</h1>
       <!-- ADD Your Space-Objects Here -->
       <img id='davids_planet' src='../assets/img/planet.png' class='animated bounce infinite'>
 
-      <img id='davids_spaceship' src='../assets/img/spaceship.gif' class=''>
 
+        <img id='davids_spaceship' src='../assets/img/spaceship.gif' class=''>
```
Comment (DavidMatthewFraser) [inline challenges/challenge2.html:44]: We should only really indent if it denotes a parent/child relationship

**Pull request merged**